package.json

```diff
@@ -52,6 +53,7 @@
     "tslint": "~6.1.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "~4.0.2"
+    "typescript": "~4.0.2",
+    "webdriver-manager": "^12.1.8"
```

Comment:
Implemented as a temporary workaround due to version mismatch of chromedriver in E2E test.